clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -413,7 +434,7 @@ public class ConsumerConfig extends AbstractConfig {
                                         HEARTBEAT_INTERVAL_MS_DOC)
                                 .define(PARTITION_ASSIGNMENT_STRATEGY_CONFIG,
                                         Type.LIST,
-                                        Arrays.asList(RangeAssignor.class, CooperativeStickyAssignor.class),
+                                        PARTITION_ASSIGNOR_DEFAULT_VALUE,
```

Comment:
This constant was introduced to be reused and that made sense, but seems it's only used here now right? If so I think keeping the default explicitly here as it was is more convenient, it just makes it easier to discover (I personally navigate to this definition a lot looking for defaults :))